### DISCUSSION TITLE
Alexa suddenly saying device is unresponsive, though they are online.

### DISCUSSION THREAD

**Discussion by mano1979** [arendst/Tasmota#11413] [Q&A] [answered]
A couple of days ago my modem/router rebooted itself. Since then some of my tasmota devices (hue emulation) cannot be controlled with alexa anymore. They are all online. I can control them through the tasmota control interface, but alexa only sees 2 or 3 of the 6 devices. All other smart products in the house work fine. 

I removed one of the tasmota devices from the Alexa app, but when i try to re-discover new devices, she says that no new devices are found.

I know there was an update of the alexa firmware just a couple of hours before the modem rebooted itself. Could it have something to do with that? Is tasmota no longer supported

**Comment by Jason2866** [comment]
Which Alexa device do you have and which generation?

**Reply by mano1979** [reply]
I have 4 echo dot 2nd gen and two echo show 1st gen

**Comment by Jason2866** [comment]
Try `SetOption109 1`
0 = (default) / 1 = force gen1 Alexa mode, for Echo Dot 2nd gen devices only

**Reply by mano1979** [reply]
i tried that allready. doesn't work either.

**Comment by mano1979** [comment]
anyone any other suggestions? No one else has this issue?

**Comment by Jason2866** [comment, marked as answer]
Try latest development version. There was a small fix.

**Reply by mano1979** [reply]
ok thank you. will try that

**Reply by mano1979** [reply]
can you give me the OTA link to that update? If i try to install the latest development version on the OTA page i get a "minimal version" error.

**Reply by ascillato** [reply]
http://ota.tasmota.com/tasmota/tasmota.bin.gz

**Reply by pkkrusty** [reply]
Aaaand now everything is working fine.  Nothing changed on my end, all devices (Wemo emulation on 9.2 and 9.3.1, and Hue emulation on 9.3.1.2) are communicating with Alexa again.  I don't get it.

**Comment by mano1979** [comment]
Thanks! that latest development version fixed it! I had devices that worked fine before, but since Alexa's latest update they couldn't be discovered anymore. With this fix they can be used again. (although amazon added/removed stuff to the firmware that i really don't like. I had routines like "evening mode". Mode's are not supported anymore. I also had a routine listening to "good night" to turn of everything. Doesn't work anymore either. she now keeps wishing me a good night, like i want that hag talking when i'm tired and want to go  to bed.....

Anyway, thank you so much for your help guys! We have been sitting in the dark for a couple of days...

**Comment by pkkrusty** [comment]
I've had an issue with Alexa not seeing my tasmota devices the last two days. I have devices with 9.2, 9.3.1, and 9.3.1.2 (which solved the multiple relay Hue emulation issue a few days ago). None of the devices are seen now. When I run discovery on the 2nd Gen Echo Dot, I see the response packets from each Tasmota device. All devices have SetOption109 enabled.

**Reply by mano1979** [reply]
Allthough i haven't looked into it yet, i saw that my amazon echo's received another update (4 days after the last one ?!?) And one of my freshly updated devices stopped working again.

Maybe they fixed the issue that broke the hue emulation support in the first place, and now the hotfix in tasmota doesn't work anymore?

**Reply by pkkrusty** [reply]
If that was the case, I would expect the Wemos emulated devices to keep working, since they never had an issue. But they are not reachable either. For me.

**Reply by CarlosDalk** [reply]
Hello. I have the same problem. All devices I have with Hue Bridge emulation no longer work with Alexa.

**Reply by mano1979** [reply]
@CarlosDalk have you updated the firmware to the latest development version?

**Reply by CarlosDalk** [reply]
Yes. I upgraded to version 9.3.1. I have an Echo Dot 3 in my house and an Echo Dot 4 here in my workshop. None find devices with Hue Bridge. Only those with WeMo.

**Reply by mano1979** [reply]
That is your problem. You did not update to the development version but to the latest release version. You need to update to development version 9.3.1.2

**Reply by CarlosDalk** [reply]
Where can I find the file?

**Reply by ascillato** [reply]
> You need to update to development version 9.3.1.2

> Where can I find the file?

@CarlosDalk

Official Latest Tasmota Development Version http://ota.tasmota.com/tasmota/tasmota.bin.gz

The whole set of firmware's types can be found at http://ota.tasmota.com/tasmota

**Comment by mano1979** [comment]
There is a link in one of the posts above here

**Reply by CarlosDalk** [reply]
Thanks. I found. I'll try...

**Reply by CarlosDalk** [reply]
Thank you so much ! It worked. I had to edit the link and find the BR version of the firmware. But it worked.
